sections/locations.liquid

```diff
@@ -108,11 +108,25 @@
           </span>
 
           <address class="tabs__address">
-            {{- address | default: "Street" -}},
-            {{ city | default: "City" }},
-            {{ region | default: "Region" }},
-            {{ zipcode | default: "ZIP code" }},
-            {{ country | default: "Country" }}
+            {%- if address != blank or city != blank or region != blank or zipcode != blank or country != blank -%}
+              {%- if address != blank -%}
+                {{ address | append: ', '}}
```

Comment:
I think that this would be the same, but shorter 🤔 

```suggestion
                {{ address }},
```

Reply:
The problem is that we need a whitespace after the `,`, otherwise the address will be squished. Doing
```
{ address }}, 
```
is quite error prone (hint: there's a space after the comma).